src/adapters/http/index.js

```diff
@@ -316,26 +316,39 @@ function HttpPouch(opts, callback) {
 
     if (typeof supportsBulkGet !== 'boolean') {
       // check if this database supports _bulk_get
-      doBulkGet(function (err, res) {
-        /* istanbul ignore else */
-        if (err) {
-          var status = Math.floor(err.status / 100);
+      self.info().then(function (info) {
```

Comment:
no `catch()` on this promise means if `info()` fails it will get swallowed. :) I suggestion just adding `.catch(callback)`